packages/runtime/src/interpreter/opcode-list.ts

```diff
@@ -1982,6 +1982,9 @@ export class GetAssetDef extends Op {
         case "AssetDecimals":
           value = BigInt(AssetDefinition.decimals);
           break;
+        case "AssetDefaultFrozen":
+          value = AssetDefinition["default-frozen"] ? 1n : 0n;
+          break;
```

Comment:
Was it missing before? We probably need to update the test as well for `GetAssetDef` Op, right?

Reply:
no, previoulsy we were considering default-frozen as string but it is boolean

Reply:
ok